### ISSUE TITLE
Drawing regular polygons page

### ISSUE THREAD

**Issue by brianharvey** [cs10/bjc-r#184]
The animation is /very/ cute, but perhaps misleading; it might suggest, for example, that an irregular polygon is one that isn't simple.  I think you can save the cuteness but also make the serious point by having four cases shown instead of two:
1. regular pentagon
2. existing irregular shape
3. square
4. rectangle

Also, the question in the text just below that animation has a typo that makes it ungrammatical.

**Comment by xtitter** [comment]
yeah, cuteness overcame better pedagogical judgement.  What constitutes a regular polygon just isn't important to this lesson, IMO.

**Comment by brianharvey** [comment]
Well, it's a /little bit/ important because if an exercise says "draw a regular pentagon" you have to know that drawing home plate won't do.

**Comment by xtitter** [comment]
The proper definition is the first sentence of the page (which used to be all there was).    Eh, I'll leave it open and assign it to you :).
